dbms/src/Storages/DeltaMerge/RowKeyFilter.h

```diff
@@ -128,20 +130,19 @@ inline Block filterUnsorted(const RowKeyRanges & rowkey_ranges, Block && block,
     size_t rows = block.rows();
     auto rowkey_column = RowKeyColumnContainer(block.getByPosition(handle_pos).column, rowkey_ranges[0].is_common_handle);
 
-    IColumn::Filter filter(rows);
-    size_t passed_count = 0;
+    IColumn::Filter filter(rows, 0);
```

Comment:
Can you add ut for `DMRowKeyFilterBlockInputStream` filtering sorted and unsorted blocks?

Reply:
there are some tests in `dbms/src/Storages/DeltaMerge/tests/gtest_dm_utils.cpp`